src/webots/engine/WbSimulationCluster.cpp

```diff
@@ -467,6 +464,17 @@ void WbSimulationCluster::handleCollisionIfSpace(void *data, dGeomID o1, dGeomID
   }
 }
 
+void WbSimulationCluster::warnMoreContactPointsThanContactJoints() {
+  static std::mutex mutex;
+  std::lock_guard<std::mutex> lock(mutex);
```

Comment:
In #5796 and in the rest of the Webots code, we are using the Qt mutexes (QMutex). Is there a particular reason for using std::mutex here?

Reply:
I just hadn't done any Qt programming before so was going with what I knew. I've switched to the Qt wrappers.